packages/cli-lib/github.js

```diff
@@ -7,6 +9,11 @@ const { extractZipArchive } = require('./archive');
 const { GITHUB_RELEASE_TYPES } = require('./lib/constants');
 const { DEFAULT_USER_AGENT_HEADERS } = require('./http/requestOptions');
 
+const GITHUB_AUTH_HEADERS = {
+  authorization:
+    global && global.githubToken ? `Bearer ${global.githubToken}` : '',
+};
```

Comment:
Does it make any difference to send and empty `authorization` header? I'm guessing it doesn't or even possibly gets stripped out but figured I'd ask

Reply:
^^ I have the same question @miketalley

Reply:
Ah good call. This should be `null` -- will fix.